src/com/gmail/chernii/oleksii/list/MyArrayList.java

```diff
@@ -0,0 +1,64 @@
+package com.gmail.chernii.oleksii.list;
+
+import com.gmail.chernii.oleksii.list.List;
+
+import java.util.Arrays;
+
+/**
+ * Created by Space on 05.03.2019.
+ */
+public class MyArrayList<E> implements List<E> {
+    private E[] elements;
+    private int length;
+    private int size;
+
+    public MyArrayList() {
+        this.length = 10;
+        this.size = 0;
+        elements = (E[]) new Object[this.length];
+    }
+
+    @Override
+    public void add(E item) {
+        if (size < length - 1) {
+            elements[size++] = item;
+        } else {
+            length *= 2;
+            elements = Arrays.copyOf(elements, length);
+            elements[size++] = item;
+        }
+    }
+
+    @Override
+    public E get(int index) {
+        if (index < 0 || index > size - 1 || size == 0) {
+            return null;
```

Comment:
You arrayList could contain a null element. So, we could find if get return as null is it the element or not right index.  
Better to throw a custom exception.